sys/include/net/gnrc/rpl.h

```diff
@@ -249,12 +249,18 @@ extern "C" {
 /**
  * @brief   Number of implemented Objective Functions
  */
-#define GNRC_RPL_IMPLEMENTED_OFS_NUMOF (1)
+#define GNRC_RPL_IMPLEMENTED_OFS_NUMOF (2)
```

Comment:
This could easily go out of hand if there come more and gives little room for optimizing out objective functions. Could the `objective_functions` array maybe become a list?